README.md

```diff
@@ -1,45 +1,146 @@
 # Catapult-explorer
 
-## Setup Client
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+
+A catapult explorer is a browser for the NEM2 blockchain, user can use it to view and track transactions, accounts, assets, and blocks.
+
+## Architecture
+
+In Catapult Explorer, `Client` is handle all fronted development such as UI / UX. `Server` is handle API / SDK request from Catapult Network.
+
+### Client
+
+#### Dependencies
```

Comment:
I'd recommend you not to list dependencies, these can be found in pakage.json.

Reply:
i think i misunderstand your previous comment 

```
client
[describe package responsability]

server
[describe package responsibility]
```

Reply:
[describe package responsability] --> what the package does.  For example, client handles the UI / UX of the explorer, where a server handles the API / SDK request from Catapult Network.